projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -252,6 +252,7 @@ private static function stats_array_to_string( $kvs ) {
 	 */
 	private static function is_amp_request() {
 		$is_amp_request = ( function_exists( 'amp_is_request' ) && amp_is_request() );
+		$is_amp_request = $is_amp_request || ( function_exists( 'ampforwp_is_amp_endpoint' ) && ampforwp_is_amp_endpoint() );
```

Comment:
This make sense. I wonder if maybe we could add comments for each line, so it's clear why we have 2 checks: one for the AMP plugin, one for the AMP for WP – Accelerated Mobile Pages plugin.

```suggestion
		// Check for AMP requests in the official AMP plugin.
		$is_amp_request = ( function_exists( 'amp_is_request' ) && amp_is_request() );
		// Check for AMP requests when using the AMP for WP – Accelerated Mobile Pages plugin.
		$is_amp_request = $is_amp_request || ( function_exists( 'ampforwp_is_amp_endpoint' ) && ampforwp_is_amp_endpoint() );
```

Reply:
@jeherve Sorry I set the PR to auto merge since the issue was kinda emergent - I'll try to add some comments in my next Jetpack PR.